lib/go-tc/users.go

```diff
@@ -105,6 +348,33 @@ type UserCurrentV40 struct {
 	LastAuthenticated *time.Time `json:"lastAuthenticated" db:"last_authenticated"`
 }
 
+// UserCurrentV50 represents the structure for the "current" user, and has the "Role" field as a *string, as opposed to a *int found in the older versions
+type UserCurrentV50 struct {
+	UserName          *string    `json:"username"`
+	LocalUser         *bool      `json:"localUser"`
+	AddressLine1      *string    `json:"addressLine1"`
+	AddressLine2      *string    `json:"addressLine2"`
+	City              *string    `json:"city"`
+	Company           *string    `json:"company"`
+	Country           *string    `json:"country"`
+	Email             *string    `json:"email"`
+	FullName          *string    `json:"fullName"`
+	GID               *int       `json:"gid"`
+	ID                *int       `json:"id"`
+	NewUser           *bool      `json:"newUser"`
+	PhoneNumber       *string    `json:"phoneNumber"`
+	PostalCode        *string    `json:"postalCode"`
+	PublicSSHKey      *string    `json:"publicSshKey"`
+	Role              *string    `json:"role"`
+	StateOrProvince   *string    `json:"stateOrProvince"`
+	Tenant            *string    `json:"tenant"`
+	TenantID          *int       `json:"tenantId"`
+	Token             *string    `json:"-"`
+	UID               *int       `json:"uid"`
+	LastUpdated       *time.Time `json:"lastUpdated"`
+	LastAuthenticated *time.Time `json:"lastAuthenticated"`
```

Comment:
`UserV50` has non-pointer types for `LastUpdated`, `LastAuthenticated`, `Role`, and `UserName`, and I think those ought to match here

Reply:
Also same as above RE: V5 alias